### PR TITLE
migrate to webpack

### LINKED ISSUE #22: webpack with babelify
We should moe to webpack with babelify and eslint ;-)

### PR DESCRIPTION
**Pull request by DanielRuf**
Fixes #22 

Please check my changes.

### CODE REVIEW

**Comment by DanielRuf** [comment]
I guess hound is not configured for ES6 / Node.

**Comment by tomscholz** [comment]
I will into changing the linter to eslint

File: webpack.dev.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            sourceMap: true,
+          },
+        }, {
+          loader: 'postcss-loader?parser=postcss-scss',
+          options: {
+            sourceMap: true,
+            ident: 'postcss',
+            plugins: [
+              require('postcss-import')(),
+              require('stylelint')(),
```
[the remark this answers is not in the document]
Comment (DanielRuf) [reply webpack.dev.js]: Because PostCSS already uses stylelint. https://github.com/postcss/postcss-loader#stylelint

**Comment by tomscholz** [comment]
Okay, this looks good so far, but when I run this on my machine I get a bunch of stylelint errors. I guess the config isn't loaded. Can you reproduce that?

**Comment by DanielRuf** [comment]
Warnings, not errors ;-)
These are actual stylelint violations. See https://github.com/stylelint/stylelint-config-standard/blob/master/index.js

**Comment by DanielRuf** [comment]
Ah now I see, will delete it.

**Comment by DanielRuf** [comment]
I totally forgot. stylelint can not yet be used with the postcss-loader, see https://github.com/stylelint/stylelint/issues/2572

**Comment by DanielRuf** [comment]
Imho linting should happen in an extra npm scripts. I'll move them to native stellint and eslint scripts.

File: webpack.dev.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    ' * @license '+pkg.license+'',
+    ' * @copyright '+new Date().getFullYear()+' Tom Scholz',
+    ' * @link '+pkg.homepage+'',
+    ' **/',
+    ''
+  ].join('\n'),
+  raw: true,
+};
+
+//const cleanPaths = [''];
```
Comment (DanielRuf) [inline webpack.dev.js]: @tomscholz I guess I forgot to remove some comments. Should I remove them or keep them there for reference?

Comment (tomscholz) [reply webpack.dev.js]: You should have push permission to the `develop` branch. You can remove them if you want :)

Comment (DanielRuf) [reply webpack.dev.js]: Done. I was not at home until now.

**Pull request merged**